torch/_refs/__init__.py

```diff
@@ -3153,6 +3155,8 @@ def native_group_norm(
     mean = _maybe_convert_to_dtype(mean, input.dtype)  # type: ignore[assignment]
     rstd = _maybe_convert_to_dtype(rstd, input.dtype)  # type: ignore[assignment]
 
+    out = out.as_strided(input.shape, input.stride())
```

Comment:
Since `as_strided` will force buffer realization in the lowering phase, I think we are not completely sure any unexpected regression may be caused.

https://github.com/pytorch/pytorch/blob/4082759925a712b7cb340164d3da3a1dab372d9f/torch/_inductor/lowering.py#L966

And account that `out = out.view(input.shape)` also has special case to use `as_strided`

https://github.com/pytorch/pytorch/blob/4082759925a712b7cb340164d3da3a1dab372d9f/torch/_refs/__init__.py#L3670

I guess we should also add it as a special case to use `as_strided` here.

Reply:
I added a condition to make this change only when the input tensor is a CPU tensor and channels last format since only in this case there is a non-contiguous load issue.